dev-libs/libxslt/libxslt-1.1.39.recipe

```diff
@@ -66,25 +88,47 @@ PATCH()
 
 BUILD()
 {
+	if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+		pythonArg="--without-python"
+	else
+		pythonArg="--with-python"
+	fi
 	autoreconf -fi
-	runConfigure ./configure
+	runConfigure ./configure $pythonArg
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 
-	rm $libDir/libexslt.la $libDir/libxslt.la \
-		$libDir/libexslt.a $libDir/libxslt.a
+	rm $libDir/libexslt.la $libDir/libxslt.la
+
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	$prefix/lib/python3.10/vendor-packages/libxsltmod.la
```

Comment:
This is missing an `rm` at the beginning.

Reply:
ow bugger 👍

Reply:
Thanks, fixed in: https://github.com/haikuports/haikuports/pull/9859